core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/BootstrapModuleFinder.java

```diff
@@ -59,17 +56,23 @@ public void buildModule(ModuleSpec.Builder builder, ModuleLoader delegateLoader)
 
             ApplicationEnvironment env = ApplicationEnvironment.get();
 
+            PathFilter filter = new PathFilter() {
+                @Override
+               public boolean accept(String path) {
+                    return path.endsWith("module.xml");
```

Comment:
Should probably be `/module.xml`?

Reply:
Ok, just in case someone wants to name something like blablahfoofoomodule.xml